module/AreaMaster.cpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file   AreaMaster.cpp
+ * @brief  エリアを攻略するクラス
+ * @author keiya121
+ */
+
+#include "AreaMaster.h"
+
+using namespace std;
+
+AreaMaster::AreaMaster(Area _area, bool _isLeftCourse, bool& _isLeftEdge, int _targetBrightness)
+  : area(_area),
+    isLeftCourse(_isLeftCourse),
+    isLeftEdge(_isLeftEdge),
+    targetBrightness(_targetBrightness)
+{
+}
+
+void AreaMaster::run()
+{
+  const int BUF_SIZE = 128;
+  char buf[BUF_SIZE];  // log用にメッセージを一時保持する領域
```

Comment:
```suggestion
  char buf[SMALL_BUF_SIZE];  // log用にメッセージを一時保持する領域
```
Logger.h に SMALL_BUF_SIZE, LARGE_BUF_SIZE が128, 256 でそれぞれ定義されてるはずだから、そっちが使えるといいかな